### ISSUE TITLE
Logo/Title remain tabable despite being set to display:none

### ISSUE THREAD

**Issue by ehowey** [ehowey/gatsby-theme-catalyst#729]
**Describe the bug**
You can set the logo or site title to display or not display based on configs in the theme options. Right now this works well visually, but for a11y the site logo remains accessible through tabbing which leads to some confusion about where the tabIndex has gone since it is not visible.

**Comment by ehowey** [comment]
Fixed by #877
